demo/src/routes/+page.svelte

```diff
@@ -16,81 +15,98 @@
-					class="btn-doc d-flex align-items-center justify-content-center"
+					class="btn btn-outline-primary rounded-pill d-flex align-items-center justify-content-center px-3"
 					aria-label="link to angular examples"
 					title="Svelte"
 				>
-					<Svg svg={svelteLogo} className="pres-card-logo me-3" />
+					<Svg svg={svelteLogo} className="pres-card-logo me-2" />
 					Svelte
 				</a>
 			</div>
 		</div>
 	</div>
 </div>
-<footer class="au-footer pt-5 mt-5 flex-grow-1">
-	<div class="container-xxl">
-		<div class="cards row align-content-around">
-			<LandingCard title="Framework agnostic">
-				A framework-agnostic core of widget factories that provide reactive models and APIs to drive the data. You can use it with any front-end
-				framework.
-			</LandingCard>
-			<LandingCard title="Headless widgets">
-				Each widget is provided in headless mode in Angular, React and Svelte using the same factory. You can use them with any front-end CSS
-				framework.
-			</LandingCard>
-			<LandingCard title="Bootstrap widgets">
-				Highly customizable Bootstrap CSS widgets are available for Angular, React and Svelte in separated packages.
-			</LandingCard>
-			<LandingCard title="Consistent">Widgets are provided for Angular, React and Svelte with the exact same markup and functionalities.</LandingCard>
-			<LandingCard title="Highly configurable">
-				Default values for all widgets props are configurable at any point of the application subtree.
-			</LandingCard>
-			<LandingCard title="Fully tested">Unit tests for headless with Vitest. E2e tests for adapters with Playwright.</LandingCard>
-			<LandingCard title="Accessible">Widgets are tested for accessibility with Axe.</LandingCard>
-		</div>
+<div class="au-features container-xxl mt-5">
+	<div class="cards row align-content-around">
+		<LandingCard title="Framework agnostic">
+			A framework-agnostic core of widget factories that provide reactive models and APIs to drive the data. You can use it with any front-end
+			framework.
+		</LandingCard>
+		<LandingCard title="Headless widgets">
+			Each widget is provided in headless mode in Angular, React and Svelte using the same factory. You can use them with any front-end CSS framework.
+		</LandingCard>
+		<LandingCard title="Bootstrap widgets">
+			Highly customizable Bootstrap CSS widgets are available for Angular, React and Svelte in separated packages.
+		</LandingCard>
+		<LandingCard title="Consistent">Widgets are provided for Angular, React and Svelte with the exact same markup and functionalities.</LandingCard>
+		<LandingCard title="Highly configurable">
+			Default values for all widgets props are configurable at any point of the application subtree.
+		</LandingCard>
+		<LandingCard title="Fully tested">Unit tests for headless with Vitest. E2e tests for adapters with Playwright.</LandingCard>
+		<LandingCard title="Accessible">Widgets are tested for accessibility with Axe.</LandingCard>
```

Comment:
Suggestion: last card is alone on the row, shouldn't we center it ?

Reply:
not yet.